charts/langkit/values.yaml

```diff
@@ -84,12 +84,14 @@ secrets:
     name: langkit-api-secret
 
 livenessProbe:
-  tcpSocket:
+  httpGet:
+    path: /health
     port: 8000
   initialDelaySeconds: 15
   periodSeconds: 10
```

Comment:
Is this a normal interval? I was thinking more towards 30seconds. It does result in some logging output and real work. 1.0.7 would be much leaner, just checking to see if the other processes are still alive, rather than testing if their output is correct. If we wait for 1.0.7 then it feels a bit safer.